content/22.vaccines.md

```diff
@@ -489,6 +489,31 @@ Additionally, given that severe SARS-CoV-2 may be associated with a dysregulated
 While trained immunity is an interesting possible avenue to complement vaccine development efforts through the use of an existing vaccine, additional research is required to assess whether the BCG vaccine is likely to confer trained immunity in the case of SARS-CoV-2.
 
+### Viral evolution and vaccine protection
+
+With these vaccines in place, one concern is how long they will remain effective, especially as the virus continues to evolve.
+Since the start of this pandemic we have already seen multiple variants emerge: B.1.1.7 (emerged in the UK), B.1.351 (emerged in South Africa), P.1 (emerged in Brazil).
+
+Viruses evolve or mutate at different rates.
+Mutation rate is measured as the number of substitutions per nucleotide per cell infected (&mu_{s/n/c)) [@doi:10.1128/JVI.00694-10].
+_c_ is estimated based on the viral titers at the start of infection and after infection, and viral yield.
+_s/n_ is estimated based on the mutation frequency (number of mutations divided by the total number of nucleotides sequenced) and the total number of mutations detected by the sequencing method used.
+Given these two calculations we can estimate mutation rate. 
+RNA viruses tend to have mutation rates between 10^(-6) to 10^(-4) [@doi:10.1128/JVI.00694-10].
+As a reference, Influenza A virus has a mutation rate of 10^(-5) whereas the mutation rate of SARS-CoV2 is lower, with (10^(-6)) mutation rate [@doi:10.24875/BMHIM.20000183].
+This accumulation of mutations allows the virus to escape recognition by the immune system [@doi:10.1093/protein/gzw017].
+These mutations are usually located in epitopes, which are regions of the virual antigen (i.e. spike protein) that are bound by antibodies.
+
+The efficacy of vaccines depends on their ability to train the immune system to recognize the virus.
+Therefore viruses can develop resistance to vaccines through the accumulation of mutations.
+The lower mutation rate of SARS-CoV2 suggests the possibility of SARS-CoV2 vaccines having a more long-lasting effect compared to Influenza A vrius ones.
+
+The vaccines in distribution have a similar efficacy against the B.1.1.7 variant, but reduced efficacy against the B.1.351 variant [@doi:10.1056/NEJMc2102017].
+Pfizer and Moderna announced that they are working on developing a booster shot to improve efficacy against the B.1.351 variant [@url:https://www.pharmaceutical-technology.com/comment/covid-19-vaccine-effectiveness-affected-by-variants/].
+The WHO continues to monitor the emergence of variants and their impact on vaccine efficacy [@url:https://www.who.int/news-room/feature-stories/detail/the-effects-of-virus-variants-on-covid-19-vaccines].
+Previous research on computationally predicting efficacy of vaccine targeting Influenza A virus, might help complement efforts to monitor these types of viral outbreaks [@doi:10.1093/cid/ciy323].
+To adapt, future vaccines may need to include multiple strains, booster shots may be required.
```

Comment:
```suggestion
To adapt, future vaccines may need to account for multiple variants and strains of SARS-CoV-2, and booster shots may be required [@doi:10.1038/s41591-021-01230-y].
```

Reply:
I had changed this to variant, and then back to strains, and then both haha. 

The language surrounding strains and variants is a bit confusing. I tend to err on the side of caution and use variant. I think a variant is designated a strain when it exhibits differences to the wild type virus... which I guess in this case is true... If anyone else wants to chime in and correct my amendment go ahead.

Reply:
I agree this term is confusing. I tend to think of variant as referring to the SNP and the strain referring to the full virus sequence. I'm leaning towards using "variant" only to be consistent with how its being referred to and I guess its the variants that are determining these strains as being different...?

Reply:
For some reason, it seems like people use "variants" to refer to "lineages" in SARS-CoV-2 -- and then some lineages may become strains? I agree it's super confusing. @RLordan's phrasing seems pretty safe to me!

Reply:
I think we will be safe. A reviewer won't be long telling us if we are wrong! haha

Reply:
These people claim to know the difference between strains and variants:
https://jamanetwork.com/journals/jama/fullarticle/2775006#more-multimedia-tab

Reply:
Hey @ajlee21, this is the last one that has been making me put off merging! Not sure what you want to do about strains/variants/etc but definitely the grammar needs to be fixed (Ronan's suggestion does this!)